projects/common-consumption/src/lib/content-details/toc-item/toc-item.component.ts

```diff
@@ -46,6 +49,8 @@ export class TocItemComponent implements OnInit, OnChanges {
       this.isMimeTypeFilterChanged = false;
     } else if (changes.tocData) {
       this.setActiveContent();
+    } else if (changes.selectAllUnit) {
+      this.selectAll(this.selectAllUnit);
```

Comment:
Change the method name to "selectAllItems"

Reply:
done